.github/workflows/build.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     strategy:
       matrix:
         java: [
-          { 'version': '11', opts: '' },
+          { 'version': '11', opts: '-pl ''!service-discovery/eureka''' },
```

Comment:
you also need to skip the docs too, as it depends on that artifacts.

Reply:
yes, I've just seen that. But I don't understand why it fails on windows and not in ubuntu ¿?¿?

Reply:
Pretty sure it's a question of cache.